app/views/uniform/fragments/returnVariationDifferences.scala.html

```diff
@@ -24,143 +24,134 @@
 @(key: String, variation: ReturnVariationData, showChangeLinks: Boolean = true, broughtForward: Option[BigDecimal] = None)(implicit m: Messages)
 
 @cost(key: String, value: BigDecimal) = {
-    @defining(
-        key match {
-            case a if a == "own-brands-packaged-at-own-sites" || a == "packaged-as-a-contract-packer" || a == "brought-into-uk" => 1
-            case b if b == "claim-credits-for-exports" || b == "claim-credits-for-lost-damaged" => -1
-            case _ => 0
-        }
-    ) { multiplier =>
-      @format_money(value * multiplier)
-    }
+@defining(
+key match {
```

Comment:
Can you undo intelliJ formatting the code? It never plays nicely with twirl and makes this very hard to read.